src/patternfly/components/ClipboardCopy/examples/clipboard-copy-expansion-example.hbs

```diff
@@ -3,11 +3,15 @@ Editable
 {{#> clipboard-copy clipboard-copy--id="3"}}
 	{{#> clipboard-copy-group}}
     {{#> clipboard-copy-group-toggle clipboard-copy-group-toggle--attribute=(concat 'id="toggle-' clipboard-copy--id '" aria-labelledby="toggle-' clipboard-copy--id ' text-input-' clipboard-copy--id '" aria-controls="content-' clipboard-copy--id '"')}}
```

Comment:
Is there a reason we don't just add the group-toggle class and all of the attributes to the button, like we did with the group-copy button?

Reply:
I added a comment here about it: https://github.com/patternfly/patternfly-next/pull/2005#discussion_r299140165  
but i think its possible to give it a try